protocol.m

```diff
@@ -130,8 +130,25 @@
 ecModel = getKcatAcrossIsoenzymes(ecModel);
 
 % STEP 12 Get standard kcat
-% Assign an enzyme cost to reactions without gene assocation (except
-% exchange, transport and pseudoreactions)
+% Assign an enzyme cost to reactions without gene assocation. These
```

Comment:
Maybe change `an enzyme cost` to `a protein cost` to keep consistent with the manuscript.

Reply:
```suggestion
% Assign a protein cost to reactions without gene assocation. These
```